.github/workflows/container_release.yaml

```
@@ -40,9 +40,11 @@ jobs:
       - name: Install dependencies
         run: poetry install
 
+      - name: Create output folder
+        run: mkdir ./results >/dev/null 2>&1
```

Why do we need to create the results folder like this upfront? :)

---

Tbh, just to make sure it's really there. We can create it once we start creating something in that sub-directory, but that would still require us to create the folder before starting to pipe. But honestly there are probably a million more efficient ways to do anything in bash :D

---

```suggestion
        run: mkdir -p results
```

Would that be the same?

---

Jap it would ;) Let's do it our way and increase readability!